src/databricks/labs/ucx/source_code/known.json

```diff
@@ -282,6 +282,309 @@
   "cycler": {
     "cycler": []
   },
+  "databricks-arc": {
+    "arc": [],
+    "arc.autolinker": [],
+    "arc.autolinker.autolinker": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'self.spark.sql(f\"show tables from {self.catalog}.{self.schema} like '*__splink__*'\")' because its value cannot be computed"
```

Comment:
noise. please change the error code for that

Reply:
done